website/project/model.py

```diff
@@ -607,6 +609,8 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     description = fields.StringField()
     category = fields.StringField(validate=validate_category, index=True)
 
+    node_license = fields.DictionaryField()
```

Comment:
Do we want this to have a callable default e.g. `fields.DictionaryField(default=dict)` ?